tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -849,13 +853,15 @@ def fit_extra(self, hyperparameters, time_limit=None, base_model_names=None, **k
         ag_args_fit = kwargs['ag_args_fit']
         ag_args_ensemble = kwargs['ag_args_ensemble']
         excluded_model_types = kwargs['excluded_model_types']
+        X_pseudo = kwargs.get('X_pseudo', None)
+        y_pseudo = kwargs.get('y_pseudo', None)
```

Comment:
shouldn't this just be a dataframe of the same format as `train_data` argument? what is the advantage of separating it into X and y (which are not arguments of `predictor` normally)

Reply:
at lower levels like the SKLearn models level they take X and y separately. `train_data` gets split into X and y down the pipeline. I'm just doing it earlier.

Reply:
The same argument could be said for train_data. Think about some reasons why we would want both to be contained in a single DataFrame. Think from the perspective of a novice ML / pandas user.

Reply:
The one reason I could see us wanting y_pseudo to be separate is if we were doing soft labels for multiclass, but even then we could represent it via a single argument and passing in a tuple.

Reply:
but for `fit_extra()` `train_data` is already split into X and y when retrieved by `self_trainer.load_data()`. So I was following that style.